src/main/java/org/example/OrganizationContact.java

```diff
@@ -0,0 +1,33 @@
+package org.example;
+
+public class OrganizationContact extends Contact {
+    private final String name;
+    private String address;
+
+    public OrganizationContact(String phoneNumber, String name, String address) {
+        super(phoneNumber);
+        this.name = name;
+        this.address = address;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getAddress() {
+        return address;
+    }
+
+    public void setAddress(String address) {
+        this.address = address;
+    }
+
+    @Override
+    public String print() {
```

Comment:
you could use toString object method. Which can be overriden from every class and when you use println(object) it will call that. Of course if we are mandatory to force everyone who extends Contact class to override print() method then its okay as it is now.

Reply:
Yeah I mean we have to print the info of every contact at some point of time, that's why I created this print